lib/config_cat/user.ex

```diff
@@ -1,36 +1,29 @@
 defmodule ConfigCat.User do
   @enforce_keys :identifier
-  defstruct [:identifier, country: "", email: "", custom: %{}]
+  defstruct [:identifier, country: nil, email: nil, custom: nil]
```

Comment:
Did you intend to revert this earlier change?

Reply:
Yeah, I think @laliconfigcat got mistaken on this one so I thought it would be best to follow what's the Ruby SDK is also doing.

> Following ruby SDK defaults for struct initialization (https://github.com/configcat/ruby-sdk/blob/master/lib/configcat/user.rb#L10)

I talked with him about it. I believe he is ok with it.